content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -0,0 +1,211 @@
+
+{{< warning >}}
+This guide has a requirement that the user is using a [plugin root CA](/docs/tasks/security/cert-management/plugin-ca-cert/)
+and has configured Istio as an intermediate CA.
+{{< /warning >}}
+
+{{< tip >}}
+This guide is tested and validated. The Istio authors feel this guide is suitable for experimentation
+but not production. Like all alpha features, this guide is subject to change.
+{{< /tip >}}
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/getting-started/#download)
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/)
+1. Check the requirements [for Pods and Services](/docs/ops/deployment/requirements/)
+1. Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, and optionally every pod in the mesh via L3 networking if enhanced performance is desired.
+
+## Prepare the guide environment
+
+1. Set the environment variables `"${ISTIO_DIR}"`, `"${WORK_DIR}"`, your cluster
+    name, and the service namespace. Ensure `"${WORK_DIR}"` is prefixed with `"${HOME}"`
+    (e.g. `WORK_DIR="${HOME}"/vmintegration`).
+
+    {{< text bash >}}
+    $ ISTIO_DIR="<the directory containing an unarchived version of Istio>"
+    $ CLUSTER_NAME="<the name of your cluster>"
+    $ SERVICE_NAMESPACE="<the name of your service namespace>"
+    $ WORK_DIR="<a certificate working directory>"
+    {{< /text >}}
+
+1. Create the `"${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"` working directories.
+
+    {{< text bash >}}
+    $ mkdir -p "${WORK_DIR}"/"${CLUSTER_NAME}"/${SERVICE_NAMESPACE}"
+    {{< /text >}}
+
+## Create certificates for use with the virtual machine and Istio control plane
+
+1. Execute the following commands to create certificates and install them in your
+    cluster for use by Istio. See
+    [Certificate Authority (CA) certificates](/docs/tasks/security/cert-management/plugin-ca-cert/)
+    for more details on configuring an external CA. The `NAME` variable is
+    used during certificate generation to uniquely identify clusters. The
+    `NAMESPACE` variable identifies the namespace where the virtual machine
+    connectivity is hosted.
+
+    {{< text bash >}}
+    $ cd "${WORK_DIR}"
+    $ make -f "${ISTIO_DIR}"/samples/certs/Makefile NAME="${CLUSTER_NAME}" NAMESPACE="${SERVICE_NAMESPACE}" "${CLUSTER_NAME}"-certs-wl
+    $ kubectl create namespace istio-system
+    $ kubectl create secret generic cacerts -n istio-system \
+        --from-file="${WORK_DIR}"/"${CLUSTER_NAME}"/ca-cert.pem \
+        --from-file="${WORK_DIR}"/"${CLUSTER_NAME}"/ca-key.pem \
+        --from-file="${WORK_DIR}"/"${CLUSTER_NAME}"/root-cert.pem \
+        --from-file="${WORK_DIR}"/"${CLUSTER_NAME}"/cert-chain.pem
+    {{< /text >}}
+
+## Install the Istio control plane
+
+If the Istio control plane is not already installed, or isn't installed with `.spec.values.global.meshExpansion.enabled=true`, you
+must enable.
```

Comment:
must enable what?

Here's a potential rewording of this:

The Istio control plane must be installed with virtual machine integration enabled (values.global.meshExpansion.enabled: true).

Here is an example IstioOperator spec that can be used to install the default control plane and configure virtual machine integration.

<example spec>

You can install this control plane spec using:

<istioctl install>

See <link to install docs> for more details on installing Istio using istioctl.

Reply:
thanks. I addressed most of these comments, although a I did the work in multiple instructions blocks rather than just one. Take a look and let me know if it looks suitable.